airbyte-integrations/connectors/source-mailchimp/source_mailchimp/spec.json

```diff
@@ -38,6 +38,12 @@
                 "type": "string",
                 "description": "An access token generated using the above client ID and secret.",
                 "airbyte_secret": true
+              },
+              "server_prefix": {
+                "title": "server_prefix",
+                "type": "string",
+                "description": "Server prefix is attached to beginning of every endpoint ping.",
+                "airbyte_secret": true
```

Comment:
Why are you adding this? The PR is adding a new stream not changing this part of the code. Please rollback the change